Generator/Sources/NeedleFramework/Parsing/FileFilters/ContentFilter.swift

```diff
@@ -16,6 +16,10 @@
 
 import Foundation
 
+/// The regex pattern that matches any component instantiation expressions
+/// with the first capture group capturing the name of the component.
```

Comment:
Maybe add a note that this really catches the instantiation of any class where the constructor has `parent` as the first argument.